tests/ProductsTest.php

```diff
@@ -73,6 +73,23 @@ public function testProductsXml()
         $this->assertXmlStringEqualsXmlString($sampleXml, $outputXml);
     }
 
+    public function testProductsDeletedXml()
+    {
+        $document = new Document('TestCatalog');
+
+        foreach (['Product', 'Variation'] as $example) {
```

Comment:
@weshooper Yeah I agree. This was just a C&P job from another function.

Ideally we'd do this, but it doesn't work because the functions return `void`:

```php
$document->addObject((new Product('PRD123'))->setDeleted());
$document->addObject((new Product('VAR123'))->setDeleted());
```